infra/modules/providers/azure/app-monitoring/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_monitor_action_group" "appmonitoring" {
+  count               = "${var.action_group_email_receiver == "" ? 0 : 1}"
```

Comment:
nifty! That's a great way to do optional resources in TF. Never seen that before.

Reply:
Haha Erik introduced me.